addons/sourcemod/scripting/szf/stocks.sp

```diff
@@ -916,3 +925,44 @@ public Action TimerKillEntity(Handle hTimer, int iEntity)
 		AcceptEntityInput(iEntity, "Kill");
 	}
 }
+
+/******************************************************************************************************/
+
+stock void ApplyVoodooCursedSoul(int iClient)
+{
+	if (!bTF2Items || TF2_IsPlayerInCondition(iClient, TFCond_HalloweenGhostMode)) return;
+
+	TF2_CreateAndEquipFakeModel(iClient, view_as<int>(TF2_GetPlayerClass(iClient)));
```

Comment:
meant to be this to fix models/crash
```TF2_CreateAndEquipFakeModel(iClient, iZombieSoulIndex[view_as<int>(TF2_GetPlayerClass(iClient))]);```

Reply:
> So after some fuck-up stuffs with raiden that i meant to test in vrlab, turns out there model problem/cause crashes, and found myself the fix for you.

I wish I was not _that_ serious when I said I was washed up, oof.


> I also notice ragdoll is having weird model stuffs, normal skin with zombie cosmetics together. Does DFS also have a fix for it? Otherwise i may have an idea i want to try.

iirc that was never a bug on DFS, this implementation is same sans the one slip-through and cleaned up code. But I can give that a look soon.